src/rdkafka_request.c

```diff
@@ -3208,6 +3208,7 @@ rd_kafka_DescribeConfigsRequest (rd_kafka_broker_t *rkb,
         if (rd_list_cnt(configs) == 0) {
                 rd_snprintf(errstr, errstr_size,
                             "No config resources specified");
+		rd_kafka_replyq_destroy(&replyq);
```

Comment:
Please use 8 whitespace as indent, not tabs.

Reply:
fixed